recipes/patsy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.4.1" %}
+{% set gh_org = "pydata" %}
+{% set gh_repo = "patsy" %}
+{% set pkg_name = "patsy" %}
```

Comment:
These two variables have the same value. Not sure it is worth the extra complexity here. (Jinja vars are handy, but abusing them makes the recipe unreadable.)